File: packages/react-native-nitro-template/android/src/main/java/com/margelo/nitro/nitroMath/HybridMath.kt
package com.margelo.nitro.nitroMath
class HybridMath : HybridMathSpec() {
    override val memorySize: Long
        get() = 0L

    override val pi: Double
        get() = Math.PI

    override fun add(a: Double, b: Double): Double {
        return a + b
    }

    override fun subtract(a: Double, b: Double): Double {
        return a - b
    }
}

File: packages/react-native-nitro-template/ios/HybridMath.swift
class HybridMath : HybridMathSpec {
  public var hybridContext = margelo.nitro.HybridContext()
  public var memorySize: Int {
    return getSizeOf(self)
  }

  public var pi: Double {
    return Double.pi
  }
  public func add(a: Double, b: Double) throws -> Double {
    return a + b
  }

    public func subtract(a: Double, b: Double) throws -> Double {
      return a - b
    }
}
